bosh-director/lib/bosh/director/instance_updater/stopper.rb

```diff
@@ -48,9 +48,6 @@ def wait_for_dynamic_drain(initial_drain_time)
       drain_time = initial_drain_time
 
       loop do
-        # This could go on forever if drain script is broken, canceling the task is a way out.
-        @config.task_checkpoint
-
```

Comment:
@cppforlife my idea is to have this check in agent_client.drain only, why to keep this one? there is a spec for this change (stopper_spec.rb, the last one on this page). not sure it's related to agent version, it will be cancelled in bosh anyway and gets an error "cancel is not supported for this action" from older agents.